packages/cms-lib/api/fileManager.js

```diff
@@ -31,21 +31,31 @@ async function getStat(portalId, src) {
   });
 }
 
-async function getFilesByPath(portalId, src) {
+async function getFiles(portalId, folderId, { offset }, showArchived) {
```

Comment:
What if we added `archived` as a param like `offset`? The reason for suggesting that approach is that it avoids a [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap).

Reply:
Thanks for pointing me to that article. I've updated this, I think its more readable now